docker/data-science/notebooks/ch12/0.setup.ipynb

```diff
@@ -61,7 +61,7 @@
     "#Get datasets\n",
     "![ ! -d 'retrotech' ] && git clone --depth=1 https://github.com/ai-powered-search/retrotech.git\n",
     "! cd retrotech && git pull\n",
-    "! cd retrotech && tar -xvf products.tgz -C '../../data/retrotech/' && tar -xvf signals.tgz -C '../../data/retrotech/'"
+    "! cd retrotech && tar -xvf products.tgz -C '../data/retrotech/' && tar -xvf signals.tgz -C '../data/retrotech/'"
```

Comment:
Todo: verify

Reply:
An error on my part

Reply:
is the `../` correct (vs. previous `../../`?

See your comment that there was an error on your part and just double checking since there was still a change.